<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>EventWidget</class>
 <widget class="QWidget" name="EventWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>919</width>
    <height>602</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Event</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QStackedWidget" name="stackedWidget">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="page1">
      <layout class="QVBoxLayout" name="verticalLayout_15">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_6">
         <item>
          <layout class="QVBoxLayout" name="verticalLayout_14">
           <item>
            <widget class="QGroupBox" name="groupBox_2">
             <property name="title">
              <string>Event description</string>
             </property>
             <layout class="QVBoxLayout" name="verticalLayout_3">
              <item>
               <widget class="QLineEdit" name="lineEditDescription"/>
              </item>
             </layout>
            </widget>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_2">
             <item>
              <widget class="QGroupBox" name="groupBox_3">
               <property name="title">
                <string>Start date</string>
               </property>
               <layout class="QVBoxLayout" name="verticalLayout_5">
                <item>
                 <widget class="DateTimeEdit" name="dateTimeEditStart">
                  <property name="enabled">
                   <bool>true</bool>
                  </property>
                  <property name="specialValueText">
                   <string/>
                  </property>
                  <property name="minimumDate">
                   <date>
                    <year>1800</year>
                    <month>1</month>
                    <day>1</day>
                   </date>
                  </property>
                  <property name="calendarPopup">
                   <bool>true</bool>
                  </property>
                 </widget>
                </item>
               </layout>
              </widget>
             </item>
             <item>
              <widget class="QGroupBox" name="groupBox_4">
               <property name="title">
                <string>End date</string>
               </property>
               <layout class="QVBoxLayout" name="verticalLayout_6">
                <item>
                 <widget class="DateTimeEdit" name="dateTimeEditEnd">
                  <property name="specialValueText">
                   <string/>
                  </property>
                  <property name="minimumDate">
                   <date>
                    <year>1800</year>
                    <month>1</month>
                    <day>1</day>
                   </date>
                  </property>
                  <property name="calendarPopup">
                   <bool>true</bool>
                  </property>
                 </widget>
                </item>
               </layout>
              </widget>
             </item>
             <item>
              <widget class="QGroupBox" name="groupBox_5">
               <property name="title">
                <string>Price list</string>
               </property>
               <layout class="QVBoxLayout" name="verticalLayout_10">
                <item>
                 <widget class="RelationalComboBox" name="comboBoxPriceList"/>
                </item>
               </layout>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <widget class="QLabel" name="labelEventUsed">
             <property name="font">
              <font>
               <bold>true</bold>
              </font>
             </property>
             <property name="styleSheet">
              <string notr="true">color: rgb(255, 0, 0);</string>
             </property>
             <property name="text">
              <string>This event is used, dates can't be changed</string>
             </property>
            </widget>
           </item>
           <item>
            <spacer name="verticalSpacer_2">
             <property name="orientation">
              <enum>Qt::Orientation::Vertical</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>20</width>
               <height>40</height>
              </size>
             </property>
            </spacer>
           </item>
          </layout>
         </item>
         <item>
          <widget class="QGroupBox" name="groupBox">
           <property name="title">
            <string>Event image</string>
           </property>
           <layout class="QVBoxLayout" name="verticalLayout_9">
            <item>
             <layout class="QVBoxLayout" name="verticalLayout_2">
              <item>
               <widget class="LabelImage" name="labelEventImage">
                <property name="text">
                 <string>NO IMAGE</string>
                </property>
                <property name="alignment">
                 <set>Qt::AlignmentFlag::AlignCenter</set>
                </property>
                <property name="hasImage" stdset="0">
                 <bool>true</bool>
                </property>
               </widget>
              </item>
              <item>
               <spacer name="verticalSpacer">
                <property name="orientation">
                 <enum>Qt::Orientation::Vertical</enum>
                </property>
                <property name="sizeHint" stdset="0">
                 <size>
                  <width>20</width>
                  <height>40</height>
                 </size>
                </property>
               </spacer>
              </item>
              <item>
               <layout class="QHBoxLayout" name="horizontalLayout">
                <item>
                 <widget class="QPushButton" name="pushButtonUpload">
                  <property name="text">
                   <string>Upload...</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QPushButton" name="pushButtonDownload">
                  <property name="text">
                   <string>Download...</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <spacer name="horizontalSpacer">
                  <property name="orientation">
                   <enum>Qt::Orientation::Horizontal</enum>
                  </property>
                  <property name="sizeHint" stdset="0">
                   <size>
                    <width>40</width>
                    <height>20</height>
                   </size>
                  </property>
                 </spacer>
                </item>
                <item>
                 <widget class="QPushButton" name="pushButtonDelete">
                  <property name="text">
                   <string>Delete</string>
                  </property>
                 </widget>
                </item>
               </layout>
              </item>
             </layout>
            </item>
           </layout>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="page2">
      <layout class="QVBoxLayout" name="verticalLayout_7">
       <item>
        <widget class="EnhancedTableView" name="tableView"/>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>EnhancedTableView</class>
   <extends>QTableView</extends>
   <header>App.Widget.View</header>
  </customwidget>
  <customwidget>
   <class>RelationalComboBox</class>
   <extends>QComboBox</extends>
   <header>App.Widget.Control</header>
  </customwidget>
  <customwidget>
   <class>LabelImage</class>
   <extends>QLabel</extends>
   <header>App.Widget.Control</header>
  </customwidget>
  <customwidget>
   <class>DateTimeEdit</class>
   <extends>QDateTimeEdit</extends>
   <header>App.Widget.Control</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>lineEditDescription</tabstop>
  <tabstop>dateTimeEditStart</tabstop>
  <tabstop>dateTimeEditEnd</tabstop>
  <tabstop>pushButtonUpload</tabstop>
  <tabstop>pushButtonDownload</tabstop>
  <tabstop>pushButtonDelete</tabstop>
  <tabstop>tableView</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
